lib/app/views/server_state.erb

```diff
@@ -6,7 +6,7 @@
         The version of the HL7 Validator currently used for validation is
         <%= Inferno::RESOURCE_VALIDATOR.version || 'not found. Please ensure that the validator is running and is up to date' %>.
       <% else %>
-        The version of the FHIR Models Validator currently used for validation is <%= FHIR::Models::VERSION %>.
```

Comment:
Is this a global s/r mistake?

Reply:
Yes, thanks. Shoot, there will probably be more of these with `FHIR::Models`.

Reply:
This was actually the only instance of this particular error.